build.gradle

```diff
@@ -61,17 +68,6 @@ subprojects {
         }
     }
 
-    def compileTasks = {
-        options.encoding = 'UTF-8'
-
-        // Nails the Java-Version of every Subproject
-        sourceCompatibility = JavaVersion.VERSION_21
-        targetCompatibility = JavaVersion.VERSION_21
-    }
-
-    compileJava(compileTasks)
-    compileTestJava(compileTasks)
-
```

Comment:
I believe this is no longer necessaray, since java 18, default encoding already is UTF-8, for all systems and jvms.

https://openjdk.org/jeps/400